src/tests/ftest/container/BasicTxTest.py

```diff
@@ -21,12 +21,13 @@
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
 '''
+from __future__ import print_function
 import os
 import time
 import traceback
 import sys
 import json
-from avocado import Test, main
+from avocado import Test
```

Comment:
(pylint-import-error) Unable to import 'avocado'

Reply:
[This](https://github.com/daos-stack/daos/blob/master/utils/check_modules.sh#L44) is the change from master you need to make these _import-error_ errors go away.  You'll get that change if you merge master to this branch and push that to this PR.